resources/migrations/000_migrations.yaml

```diff
@@ -14378,6 +14378,36 @@ databaseChangeLog:
                   constraints:
                     nullable: false
 
+  - changeSet:
+      id: v47.00-010
+      author: tsmacdonald
+      comment: Added 0.47.0 - Make metabase_table.name long enough for H2 names
+      changes:
+        - modifyDataType:
+            tableName: metabase_table
+            columnName: name
+            newDataType: varchar(256)
+      rollback:
+        - modifyDataType:
+            tableName: metabase_table
+            columnName: name
+            newDataType: varchar(254)
+
+  - changeSet:
+      id: v47.00-011
+      author: tsmacdonald
+      comment: Added 0.47.0 - Make metabase_table.display_name long enough for H2 names
+      changes:
+        - modifyDataType:
+            tableName: metabase_table
+            columnName: display_name
+            newDataType: varchar(256)
+      rollback:
```

Comment:
do we really need this rollback tho?

Reply:
The linter wanted it :shrug:

Reply:
I think you can leave an empty rollback if you like. ex,

```yaml
  - changeSet:
      id: v46.00-076
      author: noahmoss
      comment: Remove foreign key constraint on sandboxes.permission_id
      changes:
        - dropForeignKeyConstraint:
            baseTableName: sandboxes
            constraintName: fk_sandboxes_ref_permissions
      rollback:
      # Not required because a rollback would drop the `permission_id` column anyway
```

Reply:
True. By not having a rollback, the assumption we're making is that `varchar(256)` is fine for the last major version (which is almost certainly true).